packages/verification-app/src/App.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <div id="app">
+    <div :id="redesign ? 'app-new' : 'app'">
```

Comment:
I'm not sure if this id should be changed as the whole Vue-App is mounted to this id. This change is done after vue is mounted, but to be sure, I would rather use a class instead of changing the id.